### PR TITLE
Serialization and Deserialization of object.

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done.

### CODE REVIEW

**Comment by AlexanderVladimirov9090** [comment]
Done.

**Comment by AlexanderVladimirov9090** [comment]
Done.

File: serialization/src/DataClass.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Deserialize an object and return it.
+   * @param inputStream
+   * @return deserialize copy of given object
+   */
+  public Object getObject(InputStream inputStream) {
```
Comment (mgenov) [inline serialization/src/DataClass.java]: Method with common signature cannot return concrete object.

**Comment by AlexanderVladimirov9090** [comment]
You are right.

**Comment by AlexanderVladimirov9090** [comment]
Done, now I can migrate the Object in the demo.

File: serialization/src/DataClass.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Object Object = null;
```
Comment (mgenov) [inline serialization/src/DataClass.java]: Name of the references should start with lower case character.

File: serialization/src/Demo.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  public static void main(String[] args) {
+
+    Book book1 = new Book("How Not To Java", "Alexander Vladimirov", 1234589012, 111 );
+    DataClass data = new DataClass();
+
+    try {
+      FileOutputStream fileOut = new FileOutputStream("/home/clouway/developmnet/input-output/book.ser");
+      data.saveObject(fileOut, book1);
+      FileInputStream fileIn = new FileInputStream("/home/clouway/developmnet/input-output/book.ser");
+      data.getObject(fileIn);
```
Comment (mgenov) [inline serialization/src/Demo.java]: I'm not finding how you ensure that the object is read in consistent manner.

**Comment by AlexanderVladimirov9090** [comment]
Did not see that one sorry.

**Pull request merged**